unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BlockerController/Interfaces/IBlockerInstanceHandler.cs

```diff
@@ -8,7 +8,8 @@ public interface IBlockerInstanceHandler
         void DestroyAllBlockers();
         Dictionary<Vector2Int, IPoolableObject> GetBlockers();
         void HideBlocker(Vector2Int coords, bool instant = false);
-        void ShowBlocker(Vector2Int pos, bool instant = false);
+        void ShowBlocker(Vector2Int pos, bool instant = false, bool colliderEnabled = true);
+        void ToggleBlockersCollision(bool newState);
```

Comment:
```suggestion
        void SetCollision(bool newState);
```

We are not actually toggling, just setting the new collision state.